ajax/libs/d3-hexbin/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "d3-hexbin",
+  "filename": "d3-hexbin.min.js",
+  "version": "0.2.2",
+  "description": "Group two-dimensional points into hexagonal bins.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "hexbin",
+    "hexagonal",
+    "binning"
+  ],
+  "homepage": "https://d3js.org/d3-hexbin",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "http://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-hexbin.git"
+  },
+  "npmName": "d3-hexbin",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/!(*.png|package.js)"
```

```suggestion
        "*.js"
```

The only two files in the `build` directory are `d3-hexbin.js` and `d3-hexbin.min.js`, so we can use a much simpler and clearer glob here?